Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   gem "pry"
```

Comment:
OTOH, the odds that that environment still works by the time she's back are essentially nil.